src/metaswitch/clearwater/etcd_shared/plugin_utils.py

```diff
@@ -15,22 +15,21 @@
 _log = logging.getLogger("etcd_shared.plugin_utils")
 
 
-def run_command(command, namespace=None, log_error=True):
+def run_command(command_args, namespace=None, log_error=True):
     """Runs the given shell command, logging the output and return code.
 
     If a namespace is supplied the command is run in the specified namespace.
 
-    Note that this runs the provided command in a new shell, which will
-    apply shell replacements.  Ensure the input string is sanitized before
-    passing to this function.
+    Note that this runs the provided array of command arguments in a subprocess
+    call without shell, to avoid shell injection. Ensure the command is passed 
```

Comment:
Yep the other pull request https://github.com/Metaswitch/clearwater-etcd-plugins/pull/43 covers all the places calling this function.